### PR TITLE
Requestability rules grid is now Ext4

### PR DESCRIPTION
**Pull request by labkey-adam**
#### Rationale
Component was converted from Ext3 to Ext4, but this test didn't get the memo

#### Related Pull Requests
* https://github.com/LabKey/platform/pull/4959

### CODE REVIEW

File: study/test/src/org/labkey/test/tests/study/StudySimpleExportTest.java [hunk 1]
```diff
@@ -1070,14 +1071,13 @@ public void verifySpecimenSettings()
...
-        _extHelper.waitForExtDialog("Add Custom Query Rule");
-        _extHelper.selectComboBoxItem("Schema:", "auditLog");
-        _extHelper.selectComboBoxItem("Query:", "FileSystem");
-        _extHelper.selectComboBoxItem("Mark vials:", "Unavailable");
+        clickButton("Add Rule", 0);
+        click(Locator.menuItem("Custom Query"));
+        _ext4Helper.selectComboBoxItem("Schema:", "auditLog");
+        _ext4Helper.selectComboBoxItem("Query:", "FileSystem");
+        _ext4Helper.selectComboBoxItem("Mark vials:", "Unavailable");
         clickButton("Submit", 0);
```
Comment (labkey-tchad) [inline study/test/src/org/labkey/test/tests/study/StudySimpleExportTest.java]: Use helpers for menu and window to properly wait for page to change.
```suggestion
        _ext4Helper.clickExt4MenuButton(false, Ext4Helper.Locators.ext4Button("Add Rule"), false, "Custom Query");
        Window window = new Window("Add Custom Query Rule", getDriver()); // org.labkey.test.components.ext4.Window
        _ext4Helper.selectComboBoxItem("Schema:", "auditLog");
        _ext4Helper.selectComboBoxItem("Query:", "FileSystem");
        _ext4Helper.selectComboBoxItem("Mark vials:", "Unavailable");
        window.clickButton("Submit", true);
```

**Review by labkey-tchad — CHANGES_REQUESTED** [review]
Removing the synchronizing waits is likely to cause trouble on TeamCity

**Comment by labkey-adam** [comment]
> Removing the synchronizing waits is likely to cause trouble on TeamCity

Thanks for the updates. I was trying to follow the pattern in `SpecimenBaseTest.setupRequestabilityRules()`, which I don't think had these waits.

**Comment by labkey-adam** [comment]
Test is passing on this FB: https://teamcity.labkey.org/buildConfiguration/bt20/2753474?buildTab=tests

File: study/test/src/org/labkey/test/tests/study/StudySimpleExportTest.java [hunk 2]
```diff
@@ -1070,14 +1072,13 @@ public void verifySpecimenSettings()
...
-        _extHelper.clickMenuButton(false, "Add Rule", "Custom Query");
-        _extHelper.waitForExtDialog("Add Custom Query Rule");
-        _extHelper.selectComboBoxItem("Schema:", "auditLog");
-        _extHelper.selectComboBoxItem("Query:", "FileSystem");
-        _extHelper.selectComboBoxItem("Mark vials:", "Unavailable");
-        clickButton("Submit", 0);
-        _extHelper.waitForExtDialogToDisappear("Add Custom Query Rule");
-        _extHelper.selectExtGridItem("Rule", "At Repository Check", 0, "x-grid-panel", false);
+        _ext4Helper.clickExt4MenuButton(false, Ext4Helper.Locators.ext4Button("Add Rule"), false, "Custom Query");
+        Window window = new Window("Add Custom Query Rule", getDriver()); // org.labkey.test.components.ext4.Window
```
Comment (labkey-tchad) [inline study/test/src/org/labkey/test/tests/study/StudySimpleExportTest.java]: Sorry, this comment wasn't intended to be committed. I was just pointing to the component.

**Review by labkey-tchad — APPROVED** [review]

**Pull request merged**